README.md

```diff
@@ -423,14 +423,16 @@ The file format is `yaml`.
 
 The following customizations are possible:
 
-| Parameter          | Description                                                                                          |
-|--------------------|------------------------------------------------------------------------------------------------------|
-| otp\_path          | Path to the OTP installation                                                                         |
-| plt\_path          | Path to the dialyzer PLT file. When none is provided the dialyzer diagnostics will not be available. |
-| deps\_dirs         | List of directories containing dependencies. It supports wildcards.                                  |
-| apps\_dirs         | List of directories containing project applications. It supports wildcards.                          |
-| include\_dirs      | List of directories provided to the compiler as include dirs. It supports wildcards.                 |
-| otp\_apps\_exclude | List of OTP applications that will not be indexed (default: megaco, diameter, snmp, wx)              |
+| Parameter              | Description                                                                                          |
+|------------------------|------------------------------------------------------------------------------------------------------|
+| otp\_path              | Path to the OTP installation                                                                         |
+| plt\_path              | Path to the dialyzer PLT file. When none is provided the dialyzer diagnostics will not be available. |
+| deps\_dirs             | List of directories containing dependencies. It supports wildcards.                                  |
+| apps\_dirs             | List of directories containing project applications. It supports wildcards.                          |
+| include\_dirs          | List of directories provided to the compiler as include dirs. It supports wildcards.                 |
+| otp\_apps\_exclude     | List of OTP applications that will not be indexed (default: megaco, diameter, snmp, wx)              |
+| code\_reload\_enabled  | Whether or not an rpc call should be mode to a remote node to compile and reload a module            |
+| code\_reload\_node     | If code\_reload\_enabled is set to true, the node to be called should be provided. Example: "els@xx" |
```

Comment:
What about using a slightly different config format? Something like:

```
code_reload:
  node: nodename
```

This would allow us to enable/disable the feature with a single step and it would make it simpler to add extra options in the future (eg cookie, callback, etc)